chall03/npimenof.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+func main() {
+	client := http.Client{}
+	res, err := client.Get("https://chall03.hive.fi")
+	if err != nil {
+		log.Fatal(err)
+	}
+	body, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Println(string(body))
+	id := string(body)[0:strings.Index(string(body), ",")]
+	rgb := string(body)[strings.Index(string(body), ",")+1 : strings.Index(string(body), " ")]
+	rgb_separated := strings.Split(rgb, ",")
+	r, err := strconv.Atoi(rgb_separated[0][2:])
+	g, err := strconv.Atoi(rgb_separated[1][2:])
+	b, err := strconv.Atoi(rgb_separated[2][2:])
```

Comment:
you should check error right after each of them.

the 2 first could fail, and the last works, so you will keep running your code and have a problem at some point.